src/components/PlaceDescription/PlaceDescription.scss

```diff
@@ -0,0 +1,7 @@
+.content {
+  color: #32313c;
```

Comment:
what about using sass variables for color? @ryszard-put should have already extracted some

Reply:
Yes, it's of course a good idea. I replaced my colors' values with existing SASS variables.